bin/self-init.sh

```diff
@@ -132,22 +134,24 @@ if ! touch "/tmp/.kube/$KUBEMACS_HOST_KUBECONFIG_NAME"; then
 fi
 
 WILL_CREATE_CLUSTER=true
-if kind get clusters | grep "$KUBEMACS_DEFAULT_KIND_NAME" 2>&1 > /dev/null; then
-    promptEnterNoOrQuit "There appears to be a Kind cluster existing called '$KUBEMACS_DEFAULT_KIND_NAME', is it OK to delete it and recreate it?"
+if kind get clusters | grep "$KUBEMACS_KIND_NAME" 2>&1 > /dev/null; then
+    promptEnterNoOrQuit "There appears to be a Kind cluster existing called '$KUBEMACS_KIND_NAME', is it OK to delete it and recreate it?"
     clusterDeleteOrQuit=$?
     if [ "$clusterDeleteOrQuit" -eq 1 ]; then
         exit 0
     elif [ "$clusterDeleteOrQuit" -eq 2 ]; then
         WILL_CREATE_CLUSTER=false
     else
-        execPrintOutputIfFailure kind delete cluster --name "$KUBEMACS_DEFAULT_KIND_NAME"
+        execPrintOutputIfFailure kind delete cluster --name "$KUBEMACS_KIND_NAME"
     fi
 fi
 
 if [ "$WILL_CREATE_CLUSTER" = true ]; then
     echo "[status] creating kind cluster"
-    kind create cluster --name "$KUBEMACS_DEFAULT_KIND_NAME" --config /usr/share/kubemacs/kind-cluster-config.yaml
-    execPrintOutputIfFailure kubectl config set clusters.kind-"$KUBEMACS_DEFAULT_KIND_NAME".server "https://127.0.0.1:6443"
+    kind create cluster --name "$KUBEMACS_KIND_NAME" --config /usr/share/kubemacs/kind-cluster-config.yaml
```

Comment:
KUBEMACS_KIND_NAME was able to previously be set to not be prefixed with 'kind-'